core/src/main/java/org/testcontainers/lifecycle/Startable.java

```diff
@@ -0,0 +1,13 @@
+package org.testcontainers.lifecycle;
+
+public interface Startable extends AutoCloseable {
```

Comment:
Why to we need this interface in this PR? It seems it's not used anyhwere (besides being implemented).
To use it in the testframework specific implementation?

Reply:
Yes, it defines a unified set of methods for both `GenericContainer` and `DockerComposeContainer`